src/elements/stepper/stepper/stepper.ts

```diff
@@ -307,6 +313,14 @@ class SbbStepperElement extends SbbHydrationMixin(LitElement) {
     this._loaded = true;
     this.selectedIndex = this.linear ? 0 : Number(this.getAttribute('selected-index')) || 0;
     this._observer.observe(this);
+    this._resizeAbortController = new AbortController();
+    this.steps.forEach(
```

Comment:
Am I right, that if a new step is slotted that there will be no listener on it?
I propose to refactor the logic as following:
1. make sure the event bubbles
2. listen to the event on stepper level
3. get the step from the event target.

With that, the above problem should be solved.

Reply:
You could also remove the abort controller and just in the controller add the listener